lib/record_store/provider.rb

```diff
@@ -17,6 +17,8 @@ def provider_for(zone_name)
           'DNSimple'
         when /dynect\.net\z/
           'DynECT'
+        when /googledomains\.com\z/
```

Comment:
We could maybe move this to constants, but I'm also fine with the way it is 👍

Reply:
maybe it could be moved into a dictionary.

Reply:
This is the only place in the code base we have these constants. If we start using it anywhere else I'd say let's move it into constants.